recipeApp/models.py

```diff
@@ -1,3 +1,83 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+
+class Recipe(models.Model):
+            
+    class FoodType(models.IntegerChoices):
+        VEGETARIAN = 1, 'Veg'
+        NON_VEGETARIAN = 2, 'Non-Veg'
+        VEGAN = 3, 'Vegan'
+    
+        
+    class CuisineType(models.IntegerChoices):
+        SOUTH_INDIAN = 1, 'South Indian'
+        NORTH_INDIAN = 2, 'North Indian'
+        CHINESE = 3, 'Chinese'
+        
+        
+    class DifficultyLevel(models.IntegerChoices):
+        EASY = 1, 'Easy'
+        MEDIUM = 2, 'Medium'
+        HARD = 3, 'Hard'
+    
+    
+    author = models.ForeignKey(User, related_name="recipe_creator", on_delete=models.CASCADE)
```

Comment:
```suggestion
    author = models.ForeignKey(User, related_name="recipes", on_delete=models.CASCADE)
```

The related name refers to this model, not the User model.